Discord Bot/commands/get.js

```diff
@@ -5,30 +5,25 @@ module.exports = {
 	name: "get",
 	description: "Returns list of all movies in current watch list for server, or if search is specified it will attempt to search the servers list for the movie.",
 	aliases: ["list", "getmovie"],
-	execute(message, args, main, callback) {
-		var embeddedMessages = [];
-		var number = 1;
-		var description = "";
-		var searchOptions = main.searchMovieDatabaseObject(message.guild.id, "", true);
-		var movieEmbed = new MessageEmbed().setTitle("Submitted Movies").setColor("#6441a3");
-		var movie = args ? args.join(" ") : null;
+	execute(message, args, main) {
+		let embeddedMessages = [];
+		let number = 1;
+		let description = "";
+		let searchOptions = main.searchMovieDatabaseObject(message.guild.id, "", true);
+		let movieEmbed = new MessageEmbed().setTitle("Submitted Movies").setColor("#6441a3");
 
 		if (!args.length) {
 			//return to avoid hitting logic below.
-			return main.movieModel.find(searchOptions, function (error, movies) {
+			return main.movieModel.find(searchOptions, async (error, movies) => {
 				if (error) {
-					message.channel.send("Could not return list of movies, an error occured.");
-
-					return callback();
+					return message.channel.send("Could not return list of movies, an error occured.");
 				}
 				
-				if (movies.length == 0) { 
-					message.channel.send("List of unviewed movies is currently empty.");
-
-					return callback();
-				} else if (movies && movies.length > 0) {
-					for (var movie of movies) {
-						var stringConcat = `**[${number}. ${movie.name}](https://www.imdb.com/title/${movie.imdbID})** submitted by ${movie.submittedBy} on ${moment(movie.submitted).format("DD MMM YYYY")}\n
+				if (!movie || !movies.length) { 
+					return message.channel.send("List of unviewed movies is currently empty.");
+				} else {
+					for (let movie of movies) {
+						let stringConcat = `**[${number++}. ${movie.name}](https://www.imdb.com/title/${movie.imdbID})** submitted by <@${movie.submittedBy}> on ${moment(movie.submitted).format("DD MMM YYYY")}\n
```

Comment:
Love this solution but could we do something to make this backwards compatible with current data structure of submitted by? Would be nice not having to update thousands of entries in the database. 

Such as surrounding with <@{}> the submitted by string doesn't contain <@.

Reply:
Actually for simplicity sake I'll revert the changes to submittedBy after merging and re-implement them in a later release after rolling out API change for the dashboard and creating a mingration script for the mongoDB.